Patch by Ben Hutchings to fix document caching in violation
of HTTP (see http://bugs.debian.org/617713).

Index: elinks/src/config/options.inc
===================================================================
--- elinks.orig/src/config/options.inc	2017-12-01 11:07:27.761658420 +0100
+++ elinks/src/config/options.inc	2017-12-01 11:07:27.757658404 +0100
@@ -543,7 +543,7 @@
 		"cache instead of asking the server.")),
 
 	INIT_OPT_BOOL("document.cache", N_("Ignore cache-control info from server"),
-		"ignore_cache_control", 0, 1,
+		"ignore_cache_control", 0, 0,
 		N_("Ignore Cache-Control and Pragma server headers. "
 		"When set, the document is cached even with 'Cache-Control: "
 		"no-cache'.")),
